README.md

```diff
@@ -173,9 +164,14 @@ diktat {
 You can run diktat checks using task `diktatCheck` and automatically fix errors with tasks `diktatFix`.
 
 ## Run with Spotless
-[Spotless](https://github.com/diffplug/spotless) is a linter aggregator. Diktat can be run via spotless-gradle-plugin since version 5.10.0
+[Spotless](https://github.com/diffplug/spotless) is a linter aggregator.
+
+###Gradle
```

Comment:
```suggestion
### Gradle
```
Did you check how it's rendered on Github? When I hit `view file` I see that this line is not parsed as a heading

Reply:
I didn't know that. Thanks